examples/process-persistence/src/main/resources/application.properties

```diff
@@ -0,0 +1,112 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+###############
+# Persistence #
+###############
+
+# Apache KIE provides a separate Flyway subsystem that allows Kogito's internal tables to be created and migrated
+# without interfering with your application's Flyway, if you're using it.
+#
+# This property enables running this isolated Flyway mechanism automatically when the application starts.
+# If you want more control over when/how/who executes changes to your database, disable it and run the SQL scripts manually.
+kie.flyway.enabled=true
```

Comment:
Overall, I think it would be better to provide the SQL scripts and disable KIE flyway in this example, to showcase how persistence configuration is done right. But I understand that we might not want to bundle the scripts in the example itself.

If we decide to continue with the flyway approach, then I'd suggest to change the last sentence here:
`Note that the automatic schema generation using KIE Flyway is enabled to facilitate the setup of this example. For production environments, the usage of KIE Flyway is not recommended. The schema should be set up manually using the DDL scripts provided in the product's artifacts.`

Reply:
@martinweiler @josedee I think we can keep kie-flyway for simplicity in this example, but agree that we must have a note like this to clarify that this is just for development (or testing) but not for prod.